package.json

```
@@ -43,6 +43,8 @@
     "url": "https://github.com/edx/frontend-lib-special-exams/issues"
   },
   "dependencies": {
+    "@edx/frontend-build": "12.7.0",
+    "@edx/frontend-platform": "4.0.1",
```

```suggestion
    "@edx/frontend-platform": "4.1.0",
```

---

```suggestion
    "@edx/frontend-platform": "4.2.0",
```
Looks like the latest version of platform is now `4.2.0`, just a note to make sure to bump to whatever the latest is when merge conflicts are resolved. https://github.com/openedx/frontend-platform/releases